.github/ISSUE_TEMPLATE.md

```diff
@@ -1,42 +0,0 @@
-<!-- Thanks for filing an issue! Before hitting the button, please answer these questions. It's helpful to search the existing GitHub issues first. It's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of-->
-
-**Is this a request for help?**:
```

Comment:
looking forward to getting rid of this... I don't think I've ever seen someone answer "no" to "Is this a request for help?"